content/docs/next-release/operator.md

```diff
@@ -0,0 +1,666 @@
+  strategy: production
+  storage:
+    type: elasticsearch
+    options:
+      es:
+        server-urls: http://elasticsearch:9200
+  annotations:
+    key1: value1
+  resources:
+    requests:
+      memory: "64Mi"
+      cpu: "250m"
+    limits:
+      memory: "128Mi"
+      cpu: "500m"
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: kubernetes.io/e2e-az-name
+            operator: In
+            values:
+            - e2e-az1
+            - e2e-az2
+      preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 1
+        preference:
+          matchExpressions:
+          - key: another-node-label-key
+            operator: In
+            values:
+            - another-node-label-value
+  tolerations:
+    - key: "key1"
+      operator: "Equal"
+      value: "value1"
+      effect: "NoSchedule"
+    - key: "key1"
+      operator: "Equal"
+      value: "value1"
+      effect: "NoExecute"
+  volumeMounts:
+    - name: config-vol
+      mountPath: /etc/config
+  volumes:
+    - name: config-vol
+      configMap:
+        name: log-config
+        items:
+          - key: log_level
+            path: log_level
+```
+
+# Accessing the Jaeger Console (UI)
+<!-- TODO Add tabs shortcode -->
+
+## Kubernetes
+
+The operator creates a Kubernetes [`ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress/) route, which is the Kubernetes' standard for exposing a service to the outside world, but by default it does not come with Ingress providers.
+Check the [Kubernetes documentation](https://kubernetes.github.io/ingress-nginx/deploy/#verify-installation) for the most appropriate way to achieve an Ingress provider for your platform.  The following command enables the Ingress provider on `minikube`:
+
+```bash
+minikube addons enable ingress
+```
+
+Once Ingress is enabled, the address for the Jaeger console can be found by querying the Ingress object:
+
+```bash
+$ kubectl get ingress
+NAME             HOSTS     ADDRESS          PORTS     AGE
+simplest-query   *         192.168.122.34   80        3m
+```
+
+In this example, the Jaeger UI is available at http://192.168.122.34.
+
+## OpenShift
+
+When using the `operator-openshift.yaml` resource, the Operator will automatically create a `Route` object for the query services. Use the following command to check the hostname/port:
+
+```bash
+oc get routes
+```
+
+{{< info >}}
+Make sure to use `https` with the hostname/port you get from the command above, otherwise you'll see a message like: "Application is not available".
+{{< /info >}}
+
+By default, the Jaeger UI is protected with OpenShift's OAuth service and any valid user is able to login. For development purposes, the user/password combination `developer/developer` can be used. To disable this feature and leave the Jaeger UI unsecured, set the Ingress property `security` to `none` in the custom resource file:
```

Comment:
`developer/developer` doesn't work anymore with CRC. It's `kubeadmin`, with the password shown when the cluster was created.

Reply:
I would remove the entire phrase, as the previous one says it all: "any valid user is able to login".